src/WebJobs.Script.WebHost/Diagnostics/FunctionInstanceLogger.cs

```diff
@@ -83,9 +83,7 @@ internal object LogInvocationMetrics(FunctionMetadata metadata)
             {
                 string eventName = _bindingMetricEventNames.GetOrAdd(binding, static (b) =>
                 {
-                    return b.IsTrigger ?
```

Comment:
I wonder if we should retain the information we have w.r.t triggers. So triggers would be logged with a metric name that indicates it's a trigger, but all other bindings are logged w/o a direction.

Reply:
Never mind, I realize now the type will indicate that it is a trigger, since the binding type will say "Trigger"